Samples/PublicSamples/RecordingBot/src/RecordingBot.Model/Extension/ParticipantExtension.cs

```diff
@@ -56,104 +30,43 @@ public ParticipantExtension(IParticipant participant)
             }
         }
 
-        /// <summary>
-        /// Initializes a new instance of the <see cref="ParticipantExtension"/> class.
-        /// </summary>
-        /// <param name="participant">The participant.</param>
         public ParticipantExtension(Participant participant)
         {
             Resource = participant;
         }
 
-        /// <summary>
-        /// Gets the identifier.
-        /// </summary>
-        /// <value>The identifier.</value>
         public string Id { get; private set; }
 
-        /// <summary>
-        /// Gets the last modified date time of this resource.
-        /// </summary>
-        /// <value>The modified date time.</value>
         public DateTimeOffset ModifiedDateTime { get; private set; }
 
-        /// <summary>
-        /// Gets the created date time of this resource.
-        /// </summary>
-        /// <value>The created date time.</value>
         public DateTimeOffset CreatedDateTime { get; private set; }
 
-        /// <summary>
-        /// Gets the client.
-        /// </summary>
-        /// <value>The client.</value>
         [JsonIgnore]
         public ICommunicationsClient Client { get; private set; }
 
-        /// <summary>
-        /// Gets the graph client.
-        /// </summary>
-        /// <value>The graph client.</value>
         [JsonIgnore]
         public IGraphClient GraphClient { get; private set; }
 
-        /// <summary>
-        /// Gets the graph logger.
-        /// </summary>
-        /// <value>The graph logger.</value>
         [JsonIgnore]
         public IGraphLogger GraphLogger { get; private set; }
 
-        /// <summary>
-        /// Gets the resource path for this collection.
-        /// </summary>
-        /// <value>The resource path.</value>
         public string ResourcePath { get; private set; }
 
-        /// <summary>
-        /// Gets the stateful participant resource.
-        /// </summary>
-        /// <value>The resource.</value>
-        /// <exception cref="NotImplementedException"></exception>
         [JsonIgnore]
         object IResource.Resource => throw new NotImplementedException();
 
-        /// <summary>
-        /// Event fired when this resource has been updated.
-        /// </summary>
         public event ResourceEventHandler<IParticipant, Participant> OnUpdated;
 
-        /// <summary>
-        /// Deletes this participant asynchronously.
-        /// </summary>
-        /// <param name="handleHttpNotFoundInternally">If the <see cref="T:Microsoft.Graph.Communications.Calls.IParticipant" /> is already gone, whether to handle the exception gracefully or not.</param>
-        /// <param name="cancellationToken">The <see cref="T:System.Threading.CancellationToken" /> for the request.</param>
-        /// <returns>The <see cref="T:System.Threading.Tasks.Task" /> that completes after the request has been sent.
-        /// The completion of this task does not guarantee deletion. Confirmation of
```

Comment:
Not a Ghost Doc

Reply:
This information is present on the interface, and that's also where it is most important, as the comment describes the context in which this method will be called, as well as how it's implementation and behavior will be interpreted by the SDK.

This particular implementation just throws, so whatever the comment describes will be thrown out the windows anyways.

Won't readd.